### PR TITLE
Kinetic integration and dual distro compatibility

### PR DESCRIPTION
**Pull request by ipa-rmb**

### CODE REVIEW

**Comment by fmessmer** [comment]
[WIP] in the PR title should be enough to mark this PR as "Do not merge" :wink: 

don't forget to edit the PR title (remove the "[WIP] Do not merge -"-part) before merging, as the title becomes part of the commit message!

**Comment by ipa-rmb** [comment]
So, now it is fine to be merged. Travis is happy. This includes #41 .

**Review by ipa-josh — APPROVED** [review]

**Comment by ipa-rmb** [comment]
@ipa-fxm I would merge it today if you do not have objections.

**Pull request merged**
